website/addons/googledrive/tests/test_views.py

```diff
@@ -5,183 +5,83 @@
 
 from nose.tools import *  # noqa
 from framework.auth import Auth
-from website.util import api_url_for
+from website.addons.googledrive.client import GoogleDriveClient
+from website.util import api_url_for, web_url_for
 from tests.base import OsfTestCase, assert_is_redirect
-from tests.factories import AuthUserFactory, ProjectFactory
+from tests.factories import AuthUserFactory, ProjectFactory, ExternalAccountFactory
 
 from website.addons.googledrive.tests.utils import mock_folders
```

Comment:
Because this is used inside tests based on the `mock` library, it would help test readability to rename/ reorganize this, eg to `sample_folder_data`.

Actually, the entire `website.addons.googledrive.tests.utils` module could stand to be renamed: the only "utils" in that module are all unused/ candidates for deletion, so this file is basically just test data.